sass/_tables-results.scss

```diff
@@ -135,56 +138,67 @@ $results-item-border-width: 0.0625rem;
     display: block;
     float: left;
     text-decoration: none;
-    width: 30%;
+    width: 15%;
     @include clearfix;
   }
 
   .page-count.first {
-    margin-left: 30%;
+    margin-left: 15%;
   }
 
   .page-count {
+    padding: ($general-padding * 0.75);
     text-align: center;
-    width: 40%;
+    width: 20%;
 
     @include media($mobile-breakpoint) {
       font-size: 1rem;
+      padding: 0 0.2rem;
+      padding-top: ($general-padding * 0.25);
+      width: 35%;
     }
   }
 
   a:last-child {
-    float: right;
-    text-align: right;
+    text-align: center;
   }
 
   a:first-child svg {
     float: left;
   }
 
   a {
+    border: ($general-border-width / 1.5) solid $nypl-light-gray;
+    border-radius: $general-border-radius;
+    font-weight: 500;
+    padding: ($general-padding * 0.75);
+    text-align: center;
+
     svg {
-      @include nypl-icon;
+      @include nypl-icon(
+      $height: 1rem,
```

Comment:
Can you indent these two lines? Makes it a bit easier to read that it's a sass function.